divi/src/main.cpp

```diff
@@ -1540,8 +1546,11 @@ bool DisconnectBlock(CBlock& block, CValidationState& state, CBlockIndex* pindex
         // have outputs available even in the block itself, so we handle that case
         // specially with outsEmpty.
         {
+            const uint256 outputHash = (as.IsActive(Fork::SegwitLight)
+                                          ? tx.GetBareTxid() : tx.GetHash());
+
             CCoins outsEmpty;
-            CCoinsModifier outs = view.ModifyCoins(hash);
+            CCoinsModifier outs = view.ModifyCoins(outputHash);
```

Comment:
Same concern as above

Reply:
This is indeed part of "the same" change as above, so I'll close this discussion thread here (as it will always have to be in sync with the other place anyway).